go/file_manager/pfscommon/parsebody.go

```diff
@@ -0,0 +1,56 @@
+package pfscommon
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Rep interface {
```

Comment:
I did not go through the whole code so I may be wrong:
We need to be careful not to overuse interface in golang,
Please see this post for some information: https://medium.com/@cep21/preemptive-interface-anti-pattern-in-go-54c18ac0668a